source/jquery.canvaswrapper.js

```diff
@@ -384,6 +384,13 @@ don't work unless the canvas is attached to the DOM.
             position = positions[i];
             if (position.x === x && position.y === y && position.text === text) {
                 position.active = true;
+                // update the transforms
+                if (transforms) {
+                    info.element.transform.baseVal.clear();
+                    transforms.forEach(function(t) {
+                        info.element.transform.baseVal.appendItem(t);
+                    });
+                }
```

Comment:
Is there a reason we just don't do this outside the for loop (before it's called) and then remove the same code at the bottom of the function (as this doesn't seem to do anything different for each position that is iterated over)?